ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -160,6 +200,51 @@ message GetUserResponse {
     User user = 1;
 }
 
+// Required authorization: ``HasRight(ParticipantAdmin)``
+message UpdateUserRequest {
+    // The user to update.
+    //
+    // Required
+    // Modifiable
+    User user = 1;
+
+    // An update mask specifies haw and which properties of the ``User`` message are to be updated.
+    //
+    // An update mask consists of a set of update paths.
+    // An update path consists of a field path and an optional update modifier delimited by the '!' character.
+    //
+    // There are two valid update modifiers:
+    // (1) ``replace`` - replaces the current value with the new value specified in the request,
+    // (2) ``merge`` - merges the current value with the new non-default value specified in the request.
+    // The update modifier can be omitted and the default behavior is:
+    // - equivalent to the ``merge`` modifier when the update path points to a sub-message field,
+    // - equivalent to the ``merge`` modifier when the update path points to a non sub-message field, except
+    //   when the new value is the default value and then it is equivalent to the ``replace`` modifier.
```

Comment:
TODO: Maps should probably have 'merge' semantics even with the default value (the empty map)